uspv/init.go

```diff
@@ -72,8 +69,42 @@ func (s *SPVCon) Connect(remoteNode string) error {
 		if err != nil {
 			return err
 		}
+		err := Handshake(s, nil)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+func ConnectToSeedAdrs(s *SPVCon, seedAdrs []string) error {
+	// now have some IPs, go through and try to connect to one.
+	var err error
+	var connected bool
+	for i, ip := range seedAdrs {
+		log.Printf("Attempting connection to node at %s\n",
+			ip+":"+s.Param.DefaultPort)
+		s.con, err = net.Dial("tcp", ip+":"+s.Param.DefaultPort)
+		if err != nil && i != len(seedAdrs)-1 {
+			log.Println(err.Error())
+			continue
+		} else if i == len(seedAdrs)-1 {
+			connected = false
+			return fmt.Errorf(" Tried to connect to all seed Addresses from peer. Failed")
+		} else {
+			connected = true
+		}
+		if connected {
+			err := Handshake(s, seedAdrs)
+			if err != nil {
+				continue
+			}
+			break
+		}
 	}
+	return nil
+}
 
+func Handshake(s *SPVCon, seedAdrs []string) error {
```

Comment:
Here too, the way most of these things are organized is there's a big persistent struct with a bunch of state and methods on it.  (whether that's the best way to do it or not is another question :)  But might as well stick with it consistently)

Reply:
done, are the names fine?